integration_tests/test_basic.py

```diff
@@ -63,6 +64,7 @@ def test_events(cluster, suspend_capture):
         w3,
         CONTRACTS["TestERC20A"],
         key=KEYS["validator"],
+        exp_gas_used=633810,
```

Comment:
might use more gas in [older](https://github.com/crypto-org-chain/cronos/blob/18462e2960ad649e766a018b361cd2e63f4b7045/integration_tests/contracts/contracts/TestERC20A.sol#L1) version

Reply:
you are right